public/views/app.html

```diff
@@ -1,165 +1,214 @@
+    <h2 id="chat-head">CHAT</h2>
+  </div>
   <!-- Start Game Modal-->
   <div id="app-modal" ng-if="showGameModal">
-      <div id="app-modal-content">
-        <div class="modal-header">
-          <h3>Play Game</h3>
-        </div>
-        <div class="modal-body">
-          <h5> You are about to {{game.userMessage | lowercase}}. Click button to start.</h5>
-          <div id="button-div">
-            <button class= "start-button" ng-click="hideAppModal()">{{game.userMessage}}</button>
-            <button class= "start-button" ng-click="abandonGame()">Cancel</button>
-          </div>
+    <div id="app-modal-content">
+      <div class="modal-header">
+        <h3>Play Game</h3>
+      </div>
+      <div class="modal-body">
+        <h5> You are about to {{game.userMessage | lowercase}}. Click button to start.</h5>
+        <div id="button-div">
+          <button class="start-button" ng-click="hideAppModal()">{{game.userMessage}}</button>
+          <button class="start-button" ng-click="abandonGame()">Cancel</button>
         </div>
       </div>
     </div>
-    <!-- Maximum Player Reached Notification Modal-->
-    <div id="modal" ng-show="game.errorMsg">
-      <div id="modal-content">
-        <div class="modal-body" ng-if="game.errorMsg">
-            <h6>{{game.errorMsg}}</h6>
-        </div>
-        <div class="button-div">
-          <span><button class= "user-button" ><a href="/play?custom" id="startanewgame">New game</a></button></span>
-          <span><button class= "user-button" ng-click="abandonGame()">Go back</button></span>
-        </div>
+  </div>
+  <!-- Maximum Player Reached Notification Modal-->
+  <div id="modal" ng-show="game.errorMsg">
+    <div id="modal-content">
+      <div class="modal-body" ng-if="game.errorMsg">
+        <h6>{{game.errorMsg}}</h6>
+      </div>
+      <div class="button-div">
+        <span>
+          <button class="user-button">
+            <a href="/play?custom" id="startanewgame">New game</a>
+          </button>
+        </span>
+        <span>
+          <button class="user-button" ng-click="abandonGame()">Go back</button>
+        </span>
       </div>
     </div>
-    <!-- User Found Modal-->
-    <div id="modal" ng-show="game.message">
-        <div id="modal-content">
-          <button type="button" ng-click="closeModal()">&times;</button>
-          <div class="modal-body" ng-if="game.message">
-            <h6>{{game.message}}</h6>
-          </div>
-          <div class="button-div">
-            <span><button class= "user-button" ng-click="sendGameInvite()" ng-if="userInfo.user.email">Invite</button></span>
-          </div>
-        </div>
+  </div>
+  <!-- User Found Modal-->
+  <div id="modal" ng-show="game.message">
+    <div id="modal-content">
+      <button type="button" ng-click="closeModal()">&times;</button>
+      <div class="modal-body" ng-if="game.message">
+        <h6>{{game.message}}</h6>
+      </div>
+      <div class="button-div">
```

Comment:
The closing div tags seem to be more than the opening div tags. Did you addd an extra one?

Reply:
Thanks, I will check that out and fix it

Reply:
Thanks will check that out